ecl2df/common.py

```diff
@@ -55,9 +55,7 @@
     "WELSEGS",
     "WELSPECS",
 ]:
-    with open(
-        os.path.join(os.path.dirname(__file__), "opmkeywords", keyw), "r"
-    ) as file_handle:
+    with open(Path(__file__).parent / "opmkeywords" / keyw, "r") as file_handle:
         OPMKEYWORDS[keyw] = json.load(file_handle)
```

Comment:
A direct alternative when using a `Path` object is:
```suggestion
        OPMKEYWORDS[keyw] = json.load((Path(__file__).parent / "opmkeywords" / keyw).read_text())
```

Reply:
Fixed! thx.